src/main/java/org/primefaces/component/selectcheckboxmenu/SelectCheckboxMenuRenderer.java

```diff
@@ -233,6 +233,7 @@ protected void encodeMultipleLabel(FacesContext context, SelectCheckboxMenu menu
         Object valuesArray = (submittedValues != null) ? submittedValues : values;
         String listClass = createStyleClass(menu, null, SelectCheckboxMenu.MULTIPLE_CONTAINER_CLASS);
 
+
```

Comment:
Looks like you can remove this extra line break that was added?

Reply:
Sure, removed it.